### PR TITLE
Fixed typos on "Why"-page. ( varaible -> variable & its -> it's )

### PR DESCRIPTION
**Pull request by mhxbe**
Hi Basarat,

First of all: cool concept! I will definitely experiment with this soon.

I was reading the documentation and I spotted 2 typo's. They're on the **Dead CSS**-section:
https://github.com/typestyle/typestyle.github.io/blob/source/src/docs/why.md#concept-dead-css
The sixth point: **varaible** & **its**

PS: My editor stripped all trailing spaces automatically, I can modify my PR to undo those changes if that's wished.

Edited: I removed the trailing spaces edits + found some more its -> it's typos. (Thanks @notoriousb1t)

### CODE REVIEW

**Comment by notoriousb1t** [comment]
It would be good if you don't mind fixing the spaces.  Also there is another wrong "its" on line 28. If you feel like extra credit, there is that one, too 😄

File: src/docs/why.md [hunk 1]
```diff
@@ -52,20 +52,20 @@ Having the styles managed in JS (especially with TypeScript) gives you the follo
...
 Beyond that here is a boring list of additional reasons to use TypeStyle.
 
-* No context switching your brain (think its worth mentioning again).
+* No context switching your brain (think it's worth mentioning again).
 * Built in dependency system same as the rest of your JS. No special code for CSS needed.
 * Ship CSS in the same channel that you ship JS, with no special configuration being required by your library user.
 * Minification (Minify JS with existing tools). The CSS we generate is already nearly whitespace free.
 * Shared constants and reusable styles (Using variables and objects)
-* Extensible (Just use JavaScript with all its power)
+* Extensible (Just use JavaScript with all it's power)
```
Comment (notoriousb1t) [inline src/docs/why.md]: 😬 Line 68 doesn't need to be changed.  It is the possessive "its".  Otherwise, it's good 😄

Comment (mhxbe) [reply src/docs/why.md]: Ouch! I'm correcting this as we speak. 😅

**Comment by notoriousb1t** [comment]
Looks good to me

**Comment by notoriousb1t** [comment]
@mhxbe, I'm glad you are checking out our project.  Also, thank you for helping with our documentation!

**Pull request merged**